src/plugins/timeConductor/independent/independentTimeConductorPopUpManager.js

```diff
@@ -31,14 +31,17 @@ export default {
       conductorPopup: null
     };
   },
+  beforeUnmount() {
+    if (this.conductorPopup.parentNode === document.body) {
+      document.body.removeChild(this.conductorPopup);
+      this.conductorPopup = null;
+    }
+  },
   mounted() {
     this.positionBox = debounce(raf(this.positionBox), 250);
     this.timeConductorOptionsHolder = this.$el;
     this.timeConductorOptionsHolder.addEventListener('click', this.showPopup);
   },
-  beforeDestroy() {
```

Comment:
woah! nice catch-- I didn't realize we still had `beforeDestroy()` hooks in the codebase but it looks like there are a few left behind. Let's update the others to `beforeUnmounted()` while we're here